courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>pep8</li><li>Maybe add a cell where you use this function.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>